src/pages/en/guides/markdown-content.md

```diff
@@ -459,6 +459,28 @@ export default {
 }
 ```
 
+#### Markdown heading IDs
+
+Astro injects an `id` attribute into all heading elements (`<h1>` to `<h6>`) in Markdown and MDX files and provides a `getHeadings()` utility for retrieving these IDs in [Markdown exported properties](#exported-properties).
+
+You can customize these heading IDs by adding a rehype plugin that injects `id` attributes (e.g. `rehype-slug`). Your custom IDs, instead of Astro's defaults, will be reflected in HTML output and the items returned by `getHeadings()`.
+
+By default, Astro injects `id` attributes only after your rehype plugins have run. If one of your custom rehype plugins needs to access the IDs injected by Astro, you can import and use Astro’s `rehypeHeadingIds` plugin directly. Be sure to list it before any plugins that rely on it:
```

Comment:
Remaking this suggestion so I can commit it:

```suggestion
By default, Astro injects `id` attributes after your rehype plugins have run. If one of your custom rehype plugins needs to access the IDs injected by Astro, you can import and use Astro’s `rehypeHeadingIds` plugin directly. Be sure to add `rehypeHeadingIds` before any plugins that rely on it:
```

Reply:
(Turns out I’d already committed this suggestion last week 🤦 — I still like my tweaks, but seems pettier now 😅)